### PR TITLE
Added search bar to Oligos List page

### LINKED ISSUE #33: Add text searching for oligo table
Enable the user to search the oligo table using any text.

### PR DESCRIPTION
**Pull request by everyday-special**
I added a search bar on the Oligos list page. It searches Oligo names, creators, sequence, and gene locus fields. It is pretty basic, and just searches using one search term (multiple words are treated as a search phrase).

### CODE REVIEW

File: frontend/src/components/oligos/OligoList.js [hunk 1]
```diff
@@ -69,6 +75,10 @@ function OligoList(props) {
       <br />
       { isAuthenticated ? (
         <React.Fragment>
+          <form>
+            <input type='text' className='input' placeholder='Search...' name='q' />
+          </form>
+          <ul> </ul>
```
Comment (aeltanawy) [inline frontend/src/components/oligos/OligoList.js:81]: Is this line needed?

Comment (aeltanawy) [inline frontend/src/components/oligos/OligoList.js]: Is this line needed?

**Review by aeltanawy — CHANGES_REQUESTED** [review]
This is fantastic Matt!! I only have one trivial comment but this looks good to me.

Thank you so much for working on this task!

File: frontend/src/components/oligos/OligoList.js (hunk 1 above)
Comment (everyday-special) [reply frontend/src/components/oligos/OligoList.js]: It is not needed. It adds a small amount of space between the search bar and the table, which I thought looked nice. But it is purely cosmetic and can be removed. I've attached screenshots of the difference so you can let me know what you prefer.

With Space:
space

Without Space:
no space

**Review by aeltanawy — APPROVED** [review]

**Pull request merged**